OpenRA.Mods.Common/Traits/AutoTarget.cs

```diff
@@ -166,10 +180,13 @@ void Attack(Actor self, Actor targetActor, bool allowMove)
 			TargetedActor = targetActor;
 			var target = Target.FromActor(targetActor);
 			self.SetTargetLine(target, Color.Red, false);
-			attack.AttackTarget(target, false, allowMove);
+
+			foreach (var ab in attackBases)
+				if (!ab.IsTraitDisabled)
```

Comment:
Minor style nit: above you use `attackBases.Where(Exts.IsTraitEnabled)` and here you check manually.  It would be nice if you could make these consistent.